config/be-code-convention.md

```
@@ -0,0 +1,224 @@
+## DTO 네이밍
+
+- 컨트롤러 요청/응답 DTO는 `~Request`, `~Response`로 명명한다.
+- 그 외의 DTO는 `~Dto`로 명명한다.
+
+## 클래스 및 네이밍 전략
+
+- 클래스와 메서드는 카멜 케이스로 작성한다.
+- 상수는 대문자와 언더스코어를 사용한다.
+- 패키지는 모두 소문자로 작성한다.
+- 조회 메서드 네이밍
+    - get : 값을 찾지 못했을 때 반환 값이 없고 예외가 발생함. (필드 getter는 예외)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)